src/services/transfer/__tests__/transfer-out-ehr-core.test.js

```diff
@@ -101,73 +115,74 @@ describe('transferOutEhrCore', () => {
-      const error = new PresignedUrlNotFoundError();
-      const logInfoMessages = [
-        "Retrieving the patient's health record from the EHR Repository.",
-        'EHR transfer out request received'
-      ];
-
-      // when
-      getOutboundConversationById.mockResolvedValueOnce(null);
-      createOutboundConversation.mockResolvedValueOnce(undefined);
-      patientAndPracticeOdsCodesMatch.mockResolvedValue(true);
-      updateConversationStatus.mockResolvedValueOnce(undefined);
-      getEhrCoreAndFragmentIdsFromRepo.mockRejectedValueOnce(error);
-      await transferOutEhrCore({ conversationId, nhsNumber, odsCode, ehrRequestId });
-
-      // then
-      expect(updateConversationStatus).toHaveBeenCalledTimes(2);
-      expect(updateConversationStatus).toHaveBeenCalledWith(
-        conversationId,
-        ConversationStatus.OUTBOUND_STARTED
-      );
-      expect(updateConversationStatus).toHaveBeenCalledWith(
-        conversationId,
-        ConversationStatus.OUTBOUND_FAILED,
-        FailureReason.MISSING_FROM_REPO
-      );
-      expect(logInfo).toBeCalledTimes(2);
-      expect(logError).toHaveBeenCalledTimes(1);
-      expect(logInfo).toHaveBeenCalledWith(logInfoMessages[0]);
-      expect(logInfo).toHaveBeenCalledWith(logInfoMessages[1]);
-      expect(logError).toHaveBeenCalledWith(errorMessages.PRESIGNED_URL_NOT_FOUND_ERROR, undefined);
-      expect(sendCore).not.toHaveBeenCalled();
-    });
-
-    it('should stop EHR transfer if failed to download the EHR from S3 presigned URL', async () => {
-      // given
-      const error = new DownloadError();
-      const logInfoMessages = [
-        'EHR transfer out request received',
-        "Retrieving the patient's health record from the EHR Repository."
-      ];
-
-      // when
-      getOutboundConversationById.mockResolvedValueOnce(null);
-      patientAndPracticeOdsCodesMatch.mockResolvedValue(true);
-      updateConversationStatus.mockResolvedValueOnce(undefined);
-      getEhrCoreAndFragmentIdsFromRepo.mockRejectedValueOnce(error);
-      await transferOutEhrCore({ conversationId, nhsNumber, odsCode, ehrRequestId });
-
-      // then
-      expect(createOutboundConversation).toHaveBeenCalledWith(conversationId, nhsNumber, odsCode);
-      expect(patientAndPracticeOdsCodesMatch).toHaveBeenCalledWith(nhsNumber, odsCode);
-      expect(updateConversationStatus).toHaveBeenCalledWith(
-        conversationId,
-        ConversationStatus.OUTBOUND_STARTED
-      );
-      expect(logInfo).toHaveBeenCalledTimes(2);
-      expect(logInfo).toHaveBeenCalledWith(logInfoMessages[0]);
-      expect(logInfo).toHaveBeenCalledWith(logInfoMessages[1]);
-      expect(updateConversationStatus).toHaveBeenCalledWith(
-        conversationId,
-        ConversationStatus.OUTBOUND_FAILED,
-        FailureReason.EHR_DOWNLOAD_FAILED
-      );
-      expect(logError).toHaveBeenCalledWith(errorMessages.DOWNLOAD_ERROR, undefined);
-    });
+    const testCasesForCoreTransferErrors = [
```

Comment:
Missing `ERROR_CODE_06_B`?

Reply:
I can add it, but I'll explain why I missed this off:

The only place that the acknowledgementErrorCode is used in the test is to check that we're passing the gp2gpError to 'sendAcknowledgement'. 06-A and 06-B have the same gp2gpError and there's no need to update the database status so they'd just be an identical test.